Add commonly used QML modules.

StefanFabian opened the pull request:
Added the QML modules used in the examples for the [qml_ros_plugin](https://github.com/StefanFabian/qml_ros_plugin) and by a currently closed-source robot user interface using QML.

qml-module-qtmultimedia: https://packages.ubuntu.com/bionic/qml-module-qtmultimedia
qml-module-qtquick-controls: https://packages.ubuntu.com/bionic/qml-module-qtquick-controls
qml-module-qtquick-controls2: https://packages.ubuntu.com/bionic/qml-module-qtquick-controls2
qml-module-qtquick-dialogs: https://packages.ubuntu.com/bionic/qml-module-qtquick-dialogs
qml-module-qtquick-layouts: https://packages.ubuntu.com/bionic/qml-module-qtquick-layouts
qml-module-qtquick-templates2: https://packages.ubuntu.com/bionic/qml-module-qtquick-templates2
qml-module-qtquick2: https://packages.ubuntu.com/bionic/qml-module-qtquick2

StefanFabian committed:
Add commonly used QML modules.

Added the QML modules used in the examples for the [qml_ros_plugin](https://github.com/StefanFabian/qml_ros_plugin) and by a currently closed-source robot user interface using QML.

qml-module-qtmultimedia: https://packages.ubuntu.com/bionic/qml-module-qtmultimedia
qml-module-qtquick-controls: https://packages.ubuntu.com/bionic/qml-module-qtquick-controls
qml-module-qtquick-controls2: https://packages.ubuntu.com/bionic/qml-module-qtquick-controls2
qml-module-qtquick-dialogs: https://packages.ubuntu.com/bionic/qml-module-qtquick-dialogs
qml-module-qtquick-layouts: https://packages.ubuntu.com/bionic/qml-module-qtquick-layouts
qml-module-qtquick-templates2: https://packages.ubuntu.com/bionic/qml-module-qtquick-templates2
qml-module-qtquick2: https://packages.ubuntu.com/bionic/qml-module-qtquick2

tfoote commented:
Thanks for the contribution of rosdep keys. You add Ubuntu keys but no other platforms. Please take a moment to update your PR with respect to other platforms mentioned in our [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions). I would expect that there's at least debian packages available. You can add changes to the branch you have submitted from or edit the description. When you make the changes please comment back on this PR so that we get a notification that you've updated it.

StefanFabian committed:
Added debian keys for commonly used qml modules.

Debian:
qml-module-qtmultimedia: https://packages.debian.org/buster/qml-module-qtmultimedia
qml-module-qtquick-controls: https://packages.debian.org/buster/qml-module-qtquick-controls
qml-module-qtquick-controls2: https://packages.debian.org/buster/qml-module-qtquick-controls2
qml-module-qtquick-dialogs: https://packages.debian.org/buster/qml-module-qtquick-dialogs
qml-module-qtquick-layouts: https://packages.debian.org/buster/qml-module-qtquick-layouts
qml-module-qtquick-templates2: https://packages.debian.org/buster/qml-module-qtquick-templates2
qml-module-qtquick2: https://packages.debian.org/buster/qml-module-qtquick2

StefanFabian commented:
I've added them for Debian. The package names are the same.

I could find packages for arch but the structure is different and while I could locate some of the modules, I'm not comfortable with adding them without testing and I don't have arch installed.
